patroni/ctl.py

```diff
@@ -674,7 +674,8 @@ def set_defaults(config, cluster_name):
     config['postgresql'].setdefault('scope', cluster_name)
     config['postgresql'].setdefault('listen', '127.0.0.1')
     config['postgresql']['authentication'] = {'replication': None}
-    config['restapi']['listen'] = ':' in config['restapi'].get('listen', ".") or '127.0.0.1:5432'
+    config['restapi']['listen'] = (config['restapi']['listen']
+                                   if ':' in config['restapi'].get('listen', ".") else '127.0.0.1:5432')
```

Comment:
'listen' is always present in `config['restapi']`, so just ':' in config['restapi']['listen'] will work